### DISCUSSION TITLE
Clothes as containers

### DISCUSSION THREAD

**Discussion by Leander-0** [unitystation/unitystation#8414] [Ideas]
Many players had trouble with their ID cards "disappearing" out of nowhere, that happens because his jumpsuit holds both the ID and pocket slots in the UI and the clothe stores whatever is inside said slots once taken off.

it would be easy to make it like its always been that when you take off your jumpsuit all items would fall on the ground but i find it as a lazy and uncreative way to solve the problem

### Description

That is why i propose that you could access the clothes contents just like any container like the backpacks when you hold it on hand and press Z or Alt + click on it.

of course not all clothes can handle the same amount of items inside them but the idea is that said clothes store items in relation to its size, for example:

- Jumpsuits store 2 small items(pockets) slots and the ID slot.
- HoS trench coat can store 1 normal item(clothe container) and 2 small items(pockets).
- Boots, shoes and hats could store 1 VERY small item, like coins, pens or a pills.

### Justification

Being able to access the contents of the clothes not only gives a more believable sensation to the game but enriches the gameplay for everyone action wise and RP wise for example:

- As a traitor you pass a heavily guarded access point to reach your objective and the sec officers will search your backpack for lethal weapons, you anticipated this so you bough a syndicate pen full of cyanide and hide it on your hat/boot to pass the access point clean and kill your objective with said pen full of poison.

- as the clown you get arrested for being funny but you sneak up space drug pills on your shoes and decide share it with your new prison mates

**Comment by CdrCross** [comment]
I am interested in the ideas, but also hesitant. If we turn every piece of unused clothing in the game world into a container, we really are turning into a haystack for finding needles, when it comes to small piece of forensic evidence, wizzer phylacteries etc. If we go for this, I think it should go hand in hand with a cleanup of loose clothing around the station, put as much as possible in vending machines.

**Comment by PerfectTangent** [comment]
> That is why i propose that you could access the clothes contents just like any container.

@Leander-0 I mean, idk if you know this, but clothes already do have inventory slots. You can open them like any other container and add and take things from them, depending on the kinds of stuff that they allow.
